File: angular/src/app/main/manufacturing/shared/dto/mftool.dto.ts
export class MFtoolSETDto {
    tenantId: number | null;
    toolid:string;
    tooltyid:string;
    tooltypedesc:string;
    id:number;
    tooldesc: string;
    status: boolean;
    audtUser: string;
    audtDate: Date | string | null;
    createdBy: string;
    createDate: Date | string | null;
}

export class GetMFtoolSETForViewDto {
    mfToolset: MFtoolSETDto
}

export class GetMFtoolSETDtoForEditOutput {
    mftool: MFtoolSETDto
}

export class PagedResultDtoMFtoolSET {
    totalCount: number;
    items: GetMFtoolSETForViewDto[]
}

export class CreateOrEditMFtoolSETDto {
    tenantId: number | null;
    toolid:string;
    id:number;
    tooldesc: string;
    status: boolean;
    audtUser: string;
    audtDate: Date | string | null;
    createDate: Date | string | null;
}

File: aspnet-core/src/ERP.Application.Shared/Manufacturing/SetupForms/Dtos/MFTOOLDto.cs
using System;
using Abp.Application.Services.Dto;

namespace ERP.Manufacturing.SetupForms.Dtos
{
    public class MFTOOLDto : EntityDto
    {
        public int? TenantId { get; set; }

        public string TOOLID { get; set; }

        public string TOOLDESC { get; set; }

        public bool STATUS { get; set; }

        public string TOOLTYID { get; set; }

        public string AudtUser { get; set; }

        public DateTime? AudtDate { get; set; }

        public string CreatedBy { get; set; }

        public DateTime? CreateDate { get; set; }

    }
}